extensions/ql-vscode/src/interface-utils.ts

```diff
@@ -149,7 +150,7 @@ export function getHtmlForWebview(
 <html>
   <head>
     <meta http-equiv="Content-Security-Policy"
-          content="default-src 'none'; script-src 'nonce-${nonce}'; style-src ${styleSrc}; connect-src ${webview.cspSource};">
+          content="default-src 'none'; script-src 'nonce-${nonce}'; font-src ${styleSrc}; style-src ${styleSrc}; connect-src ${webview.cspSource};">
```

Comment:
🤔 I'm not sure this makes sense. Does setting `font-src '${webview.cspSource}'` work?